### PR TITLE
editing date field, predicate is now verbatimEventDate and label is '…

### LINKED ISSUE #186: METADATA: data type for the "EDTF Date" field is EDTF
**ACCEPTANCE**
 - [ ] The date field (:RDF::Vocab::DC11.date) used the ~Europeana~ Extended Date Time Format (EDTF) as it's data type
- [x] The field should have the label "EDTF Date" 

**REFERENCE**
https://docs.google.com/spreadsheets/d/1HZpXyVM8xaZZ9mE2uzsbUmWKbCd6NYJTESOrpQeMa18/edit#gid=0

### PR DESCRIPTION
**Pull request by jenlindner**
…EDTF Date'. Fixes #186.

### CODE REVIEW

File: app/views/hyrax/base/_attribute_rows.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 <%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
 <%= presenter.attribute_to_html(:keyword, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:date_created, render_as: :linked, search_field: 'date_created_desim') %>
-<%= presenter.attribute_to_html(:date) %>
+<%= presenter.attribute_to_html(:date, label: "EDTF Date") %>
```
Comment (little9) [inline app/views/hyrax/base/_attribute_rows.html.erb]: Over in epigaea, @no-reply and @mark-dce requested that we don't use labels in the views, but use the locale file instead. 

You'll need to change it in the blacklight and simple form sections of the locale file:
https://github.com/curationexperts/epigaea/blob/master/config/locales/hyrax.en.yml

File: app/views/records/edit_fields/_date.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+  <%= f.input :date, as: :multi_value, label: "EDTF Date", input_html: { class: 'form-control' }, required: f.object.required?(:date) %>
```
Comment (little9) [inline app/views/records/edit_fields/_date.html.erb]: This label is controlled with the simple_form section of the Hyrax locale file

**Review by little9 — CHANGES_REQUESTED** [review]

File: app/views/hyrax/base/_attribute_rows.html.erb (hunk 1 above)
Comment (jenlindner) [reply app/views/hyrax/base/_attribute_rows.html.erb]: Ah okay, thanks @little9.

**Comment by jenlindner** [comment]
Let me know if this is good, @little9?

**Review by little9 — APPROVED** [review]

**Pull request merged**